MultiSource/Benchmarks/Ptrdist/anagram/anagram.c

```diff
@@ -279,7 +279,7 @@ void Fatal(char *pchMsg, unsigned u) {
  * byte streams are concatenated, and terminated with a 0.
  */
 
-void ReadDict(_Ptr<char> pchFile) {
+void ReadDict(char *pchFile : itype(_Ptr<char>)) {
```

Comment:
Okay, I will remember that string pointers are allowed to stay `char *` for the moment.